mbed_flasher/flashers/FlasherMbed.py

```diff
@@ -153,14 +157,16 @@ def flash(self, source, target, method, no_reset):
             return self.try_pyocd_flash(source, target)
 
         if method == 'edbg':
-            self.logger.debug("edbg is not supported for Mbed devices")
-            return EXIT_CODE_EGDB_NOT_SUPPORTED
+            raise FlashError(message="edbg is not supported for Mbed devices",
+                             return_code=EXIT_CODE_EGDB_NOT_SUPPORTED)
 
         return retry(
             logger=self.logger,
             func=self.try_drag_and_drop_flash,
             func_args=(source, target, no_reset),
-            conditions=[EXIT_CODE_OS_ERROR])
+            conditions=[EXIT_CODE_OS_ERROR,
+                        EXIT_CODE_DAPLINK_TRANSIENT_ERROR,
+                        EXIT_CODE_DAPLINK_SOFTWARE_ERROR])
```

Comment:
Does function `retry()` parameter meant to be configurable by user, e.g. clitest?  
if so, it seems not exposed here, maybe also need changes in `flash.py`.

Reply:
It is not currently meant to be configurable by user.

Reply:
okay